e2e/cypress/integration/02_global_filter.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('global projects filter', () => {
+  before(() => {
+    cy.login('/').then(() => {
+      let admin = JSON.parse(localStorage.getItem('chef-automate-user'))
```

Comment:
hrm it's OK if that's the pattern we're currently using; but I do wonder if there's better ways to pass around state in (cypress) tests than using the browser's local storage. After all, you _just happen to know_ what is in there; it's not a public API, or local state. It could change, and it would be fine if it did change, in a way such that the UI could still work, only these tests would break because they make use of something we have no guarantees about 🤔

Reply:
I know-- seems like we should be able to use a token since these requests aren't even _using_ the UI. I wanted to get it "working" as a first pass, but I think it's worth investigating.